docs/devguide/building/index.rst

```diff
@@ -317,3 +471,62 @@ Tests
 Tests are not built by default, ``--tests`` must be passed to the configure script.
 All tests can be run using :ghfile:`tests/auto_run_tests.pl`.
 See :doc:`/internal/running_tests` for running all tests or individual tests.
+
+..
+    Sect<1.3>
```

Comment:
These comments should come after an old section header. It's missing so I don't see a point in keeping the comment.

Reply:
Should I remove all such comments?

Reply:
Probably just this one here. The idea behind them is that we or someone else could find a section using an old devguide section number.  I also used it during the conversion to help compare sections. I'm not sure that's worth the cost of keeping these comments, but I think that should be a different discussion/PR.